extensions/lima/package.json

```diff
@@ -20,6 +20,11 @@
           "default": "",
           "description": "Custom path to limactl binary (Default is blank)"
         },
+        "lima.home": {
+          "type": "string",
+          "default": "",
```

Comment:
I think you will need to set the default as the value of LIMA_HOME if defined or `~/.lima` elsewhere

As an example, how it is done for Kubernetes' Kubeconfig file: https://github.com/podman-desktop/podman-desktop/blob/main/packages/main/src/plugin/kubernetes/kubernetes-client.ts#L224-L230

Reply:
Hmm, yeah, that's reasonable

Reply:
it is not possible to define a default like that using package.json, so basically it's as in this PR. Blank = default value

Reply:
ok, my bad. LGTM